src/core/files/secrets/tpl/arkcase-server.yaml

```diff
@@ -163,3 +163,8 @@ application:
       {{- if $foia }}
       - ARKCASE_PORTAL_ADMINISTRATOR@{{ $foiaDomain }}
       {{- end }}
+      
+notification:
+  arkcase.port: {{ $baseUrl.port }}
+  arkcase.url: {{ printf "%s//%s" $baseUrl.scheme $baseUrl.hostname | quote }}
```

Comment:
Does `baseUrl.scheme` include the colon at the end?  Is it `https` or `https:`?

Reply:
@david-oc-miller, you're most likely right, the scheme is HTTPS, but for some reason is correctly rendered inside my arkcase-server,yaml. Will change it with colon and see if it will render correctly again.

Reply:
@david-oc-miller great catch !